app/api/routers/breeze_buddy/users/__init__.py

```
@@ -155,6 +157,23 @@ async def get_user_account_by_id(
     return await get_user_by_id_handler(user_id, current_user)
 
 
+@router.get("/user/{user_id}/access", response_model=UserAccessResponse)
+async def get_user_access(
+    user_id: str,
+    current_user: UserInfo = Depends(get_current_user_with_rbac),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)